docker/Dockerfile.wallet_server

```diff
@@ -53,5 +54,5 @@ ENV BANDWIDTH_LIMIT=1000000000000000000000000000000000000000000
 ENV MAX_SESSIONS=1000000000
 ENV MAX_SEND=1000000000000000000
 ENV EVENT_LOOP_POLICY=uvloop
-COPY ./lbry/scripts/wallet_server_entrypoint.sh /entrypoint.sh
+COPY ./build/wallet_server_entrypoint.sh /entrypoint.sh
```

Comment:
Is this the correct path? Didn't you rename it to `./docker`?

Reply:
yes, ty